hunting-buddy.lic

```diff
@@ -15,6 +15,7 @@ class HuntingBuddy
     args = parse_args(arg_definitions, true)
 
     @settings = get_settings
+    @bail_on_fam = @settings.stop_on_fam_drag || false
```

Comment:
❌ Please spell out the setting (`...familiar`) to help with clarity and discoverability